README.md

```diff
@@ -1 +1,157 @@
-# First-capstone-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b>First capstone Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [First Capstone Project about A musical concert] <a name="about-project"></a>
+
+This project is my First capstone project
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+This Project was built with HTML, CSS and JavaScript
+
+### Key Features <a name="key-features"></a>
+
+The website was tested on the following accessibility requirements: Title, Image text alternatives, Text headings, Color contrast, Resize, Interaction, Moving content, Multimedia, The basic structure of the page
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+ [Live Demo Link](https://www.loom.com/share/ba47dbd3d3624b328cdee21723fc69f4)
+
```

Comment:
- [ ] [OPTIONAL] Good job, you have worked hard on this, well done :clap: , Just one small suggestion, the live demo link section in your Readme is for your website's live link, putting your video presentation in it is not recommended and is also confusing, you can consider adding another section for presentation and there add the video presentation link, or just add it below the live demo link mentioning it is a video presentation but please use the live demo section for a link to your deployed site :+1: